dist/at-angular-resource.js

```diff
@@ -0,0 +1,52 @@
+/* istanbul ignore next */
+var __extends = (this && this.__extends) || function (d, b) {
+    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
+    function __() { this.constructor = d; }
+    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
+};
```

Comment:
Please set `"noEmitHelpers": true,` flag in tsconfig and add dependency or peerDependency for `ts-helpers`